src/DiveWish.spec.js

```diff
@@ -1,8 +1,9 @@
 import { render, screen } from '@testing-library/react';
+import { userEvent } from '@storybook/testing-library';
 import DiveWish from './DiveWish';
 
 describe('ListItem', () => {
```

Comment:
```suggestion
describe('DiveWish', () => {
```

Reply:
Ok, I should really start using the find&replace function